packages/gestalt/src/Badge.css

```diff
@@ -3,10 +3,17 @@
   composes: rounding1 from "./Borders.css";
   align-content: center;
   display: flex;
-  padding: var(--space-0) 6px;
   white-space: nowrap;
 }
 
+.padding {
+  padding: var(--space-0) 6px;
+}
+
+.paddingVR {
+  padding: var(--sema-space-0) var(--sema-space-100);
```

Comment:
Ideally we use component tokens for these

Reply:
we don't have cmp token for spacing yet.. i think